File: reverse-string/reverse-string.py
class Solution:
    def reverseString(self, s: List[str]) -> None:
        """
        Do not return anything, modify s in-place instead.
        """
        half = len(s) // 2
        for i in range(half):
            j = -i - 1
            s[i], s[j] = s[j], s[i]

File: reverse-string/reverse-string.js
var reverseString = function(s) {
    let length = s.length;
  let half = Math.floor(length / 2);
  for (let i = 0; i <= half; i++) {
    [s[i], s[length]] = [s[length], s[i]];
    length--;
  }
};
